rsmtool/rsmexplain.py

```diff
@@ -36,6 +36,47 @@
 from .utils.wandb import init_wandb_run, log_configuration_to_wandb
 
 
+def verify_config_features(explain_config, rsmtool_config, feature, logger=None):
+    """
+    Verify and update a specific feature in the explanation configuration.
+
```

Comment:
Add here a description mentioning that rsmtool values will override rsmexplain values.